src/common/PeopleTile/styled.js

```diff
@@ -4,7 +4,7 @@ export const StyledTile = styled.div`
   display: flex;
   flex-direction: column;
   width: 248px;
-  min-height: 339px;
+  height: 100%;
```

Comment:
Co nam to daje, bo ja po szybkim porównaniu nie widzę żadnej różnicy?

Reply:
Spójność z movies, jeśli z API idą wszystkie postery ludzi w jednym rozmiarze to nie będzie różnicy, ale jak zrobią tak jak w movies, że pójdzie jakiś inny rozmiar to bez tego się rozjedzie bo wysokość będzie się dostosowywać do treśći.

Reply:
No, ale w sumie jeśli mamy na sztywno na ten moment ustawione rozmiary zdjęć, to chyba i tak nic nie zmieni?

Reply:
A no tak, bo w Poeple Poster ma na stałe, już mi się miesza wszystko. No to tak, tutaj nic nie robi to w takim razie.

Reply:
Tia, w PeopleTile to faktycznie nie potrzebne